### PR TITLE
assertions cat.pbc

### PR DESCRIPTION
**Pull request by grewered**

### CODE REVIEW

**Comment by sachseka** [comment]
Please pay attention to checks

**Comment by grewered** [comment]
error: data frames are type character - see #25

File: R/cat.pbc.R [hunk 1]
```diff
@@ -1,5 +1,10 @@
 catPbc <- function(datRaw, datRec, idRaw, idRec, context.vars = NULL,
                    values, subunits, file.name = NULL, verbose = FALSE) {
+  lapply(c(datRaw, datRec), checkmate::assert_data_frame)
```
Comment (sachseka) [inline R/cat.pbc.R:3]: please use list(datRaw, datRec)

File: R/cat.pbc.R [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+  lapply(c(idRaw, idRec), checkmate::assert_scalar)
+  checkmate::assert_vector(context.vars, null.ok = TRUE)
+  lapply(c(values, subunits), checkmate::assert_list, types = "data.frame")
```
Comment (sachseka) [inline R/cat.pbc.R:6]: list(values, subunits)

**Review by sachseka — CHANGES_REQUESTED** [review]

**Review by sachseka — APPROVED** [review]

**Pull request merged**